lib/MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include README.md
+include Pipfile
```

Comment:
Pipfile is required because as part of our installation, we copy over all the packages we've defined in Pipfile to requirements.